product_docs/docs/epas/16/reference/sql_reference/03_functions_and_operators/07_data_type_formatting_functions.mdx

```diff
@@ -28,11 +28,11 @@ The EDB Postgres Advanced Server formatting functions provide a powerful set of
 | `TO_NCHAR(DATE  [, format ])`       | `NVARCHAR2`   | Convert a date/time to a formatted string of national character data type.                     | `TO_NCHAR(timestamp '2022-04-20 17:31:12.66', 'Day: MONTH DD, YYYY')`                                      | `Wednesday: APRIL     20, 2022`                       |
 | `TO_NUMBER(string [, format ])`     | `NUMBER`      | Convert a number formatted string to a `NUMBER` data type.                                      | `TO_NUMBER('2,412-', '999,999S')`                                                                          | `-2412`                                               |
 | `TO_TIMESTAMP(string, format)`      | `TIMESTAMPTZ` | Convert a timestamp formatted string to a TIMESTAMP WITH TIME ZONE data type.                   | `TO_TIMESTAMP('05 Dec 2000 08:30:25 pm', 'DD Mon YYYY hh12:mi:ss pm')`                                     | `05-DEC-00 20:30:25 +05:30`                           |
-| `TO_TIMESTAMP_TZ(string, format)`   | `TIMESTAMPTZ` | Convert a timestamp formatted string to a TIMESTAMP WITH TIME ZONE data type.                   | `TO_TIMESTAMP_TZ ('2003/12/13 10:13:18 -8:00', 'YYYY/MM/DD HH:MI:SS TZH:TZM')`                             | `13-DEC-03 23:43:18 +05:30`                           |
+| `TO_TIMESTAMP_TZ(string[, format])`   | `TIMESTAMPTZ` | Convert a timestamp formatted string to a TIMESTAMP WITH TIME ZONE data type.                   | `TO_TIMESTAMP_TZ ('2003/12/13 10:13:18 -8:00', 'YYYY/MM/DD HH:MI:SS TZH:TZM')`                             | `13-DEC-03 23:43:18 +05:30`                           |
```

Comment:
We should probably update the year in these examples.

Reply:
That's fine I think